react/week2/new-todo/src/Todoitem.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+
+function Todoitem({ id, description, deleteItem }) {
+  const [done, setDone] = useState(false);
+  return (
+    <div>
+      <li
+        style={
+          done
+            ? { textDecoration: "line-through" }
+            : { textDecoration: "none" }}>
+       
+        {description}
+        <span>
+          <input type="checkbox" id={id} onClick={() => setDone(!done)} />
+          <button onClick={deleteItem }>Delete</button>
```

Comment:
Delete is not working for you. Why? you are not passing id in deleteItem function. Also this delete function is executing here so you have to call it here.
<button onClick={() =>deleteItem(id) }>Delete</button>